AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs

```diff
@@ -183,9 +184,17 @@ public void ClickToProgress(By element, By newElement, int timeout = 20)
 
         public void Click(By element, int timeout = 20)
         {
-            RetryOnStaleElement(() => PerformClick(element, timeout));
+            // 09/06/2021 - Keep for now as this might need to be restored
+            //RetryOnStaleElement(() => PerformClick(element, timeout));
+            NUnit.Framework.TestContext.WriteLine($"Attempting click of element {element.ToString()} on {Driver.Url} ");
+            WebDriverWait wait = new WebDriverWait(Driver, timeout: TimeSpan.FromSeconds(30))
+            {
+                PollingInterval = TimeSpan.FromSeconds(1),
```

Comment:
Is this defaulted? Are we overriding the default to make it do it every second?

Reply:
I'm going to convert this to a helper class as it's in two classes and may be used in more later.

Reply:
Values are not defaulted so I will be creating a config object to read a file to set defaults like this.  VIH-7843